ci/pipelines/deploy-to-test.yml

```diff
@@ -684,6 +684,7 @@ jobs:
         params:
           <<: *aws_assumed_role_creds
           CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+          STUNNEL_IMAGE_TAG: 1-stunnel-release
```

Comment:
Are we just hardcoding this for the moment then?

Reply:
Yes